src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -465,22 +470,45 @@ def test_acr_image_import(self, resource_group):
         Two source images each of which stays in a different source registries mentioned above.
         '''
         self.kwargs.update({
+            'resource_id': '/subscriptions/dfb63c8c-7c89-4ef8-af13-75c1d873c895/resourcegroups/resourcegroupdiffsub/providers/Microsoft.ContainerRegistry/registries/sourceregistrydiffsub',
+            'resource_imageV1': 'sourceregistrydiffsub.azurecr.io/microsoft:azure-cli-1',
+            'resource_imageV2': 'sourceregistrydiffsub.azurecr.io/microsoft:azure-cli-2',
+            'source_registry_rg': 'resourcegroupsamesub',
+            'source_loc': 'westus',
+            'source_registry_name': source_registry_name,
             'registry_name': registry_name,
-            'rg_loc': 'eastus',
             'sku': 'Standard',
-            'resource_id': '/subscriptions/a7ee80a4-3d5e-45c2-9378-36e8d98f4d13/resourceGroups/resourcegroupdiffsub/providers/Microsoft.ContainerRegistry/registries/sourceregistrydiffsub',
-            'source_image_diff_sub': 'builder:latest',
-            'source_image_same_sub': 'sourceregistrysamesub.azurecr.io/builder:latest',
-            'source_image_same_registry': '{}.azurecr.io/builder:latest'.format(registry_name),
-            'source_image_by_digest': 'sourceregistrysamesub.azurecr.io/builder@sha256:bc3842ba36fcc182317c07a8643daa4a8e4e7aed45958b1f7e2a2b30c2f5a64f',
-            'tag_diff_sub': 'repository_diff_sub:tag_diff_sub',
+            'rg_loc': 'eastus',
+            'source_image_diff_sub': 'microsoft:azure-cli',
+            'source_image_same_sub': '{}.azurecr.io/microsoft:azure-cli'.format(source_registry_name),
+            'source_image_same_registry': '{}.azurecr.io/microsoft:azure-cli'.format(registry_name),
+            'source_image_by_digest': '{}.azurecr.io/azure-cli@sha256:2d00e582eb3bff20a7febc04222aafda5bdc58df64bc1ddf74efbaf0e51dbba5'.format(source_registry_name),
             'tag_same_sub': 'repository_same_sub:tag_same_sub',
             'tag_multitag1': 'repository_multi1:tag_multi1',
             'tag_multitag2': 'repository_multi2:tag_multi2',
             'tag_same_registry': 'repository_same_registry:tag_same_registry',
-            'tag_by_digest': 'repository_by_digest:tag_by_digest'
+            'tag_by_digest': 'repository_by_digest:tag_by_digest',
+            'source_image_public_registry_dockerhub': 'registry.hub.docker.com/library/hello-world',
```

Comment:
please us docker.io.  There is special handling for that hostname in the rp.

Reply:
Used